examples/cra-kitchen-sink/src/components/FlowTypeButton.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import React from 'react';
```

Comment:
I don't think our local `.eslintrc` has rules for flow so it's probably a good idea to disable it for this file. `/* eslint-disable */`